re.rizin.cutter.yml

```diff
@@ -0,0 +1,80 @@
+app-id: re.rizin.cutter
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.15-21.08'
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: /bin/bash
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+modules:
+  #stolen from https://github.com/flathub/org.freecadweb.FreeCAD/blob/master/org.freecadweb.FreeCAD.yaml
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      prepend-path: /usr/lib/sdk/llvm12/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTS=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.2-src/pyside-setup-opensource-src-5.15.2.tar.xz
+        sha256: b306504b0b8037079a8eab772ee774b9e877a2d84bab2dbefbe4fa6f83941418
+      - type: shell
+        commands:
+          - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp # https://bugreports.qt.io/browse/PYSIDE-787
+          - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+
+  - name: rizin
+    buildsystem: meson
+    build-options:
+      prepend-path: /usr/lib/sdk/llvm12/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+    config-opts:
+      - --buildtype=release
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.3.4/rizin-src-v0.3.4.tar.xz 
+        sha256: eea49b396387c09d19705aab02a617cdb15682fca67f101ff2b27eef94a710e9
+
+
+  - name: graphviz
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/2.49.0/graphviz-2.49.0.tar.gz
+        sha256: 17ade6189beade7c0784090f283c1f78f6b286ec38e3d2e927d3e42d625d0e8d
+
+  - name: cutter
+    buildsystem: cmake-ninja
+    build-options:
+      prepend-path: /usr/lib/sdk/llvm12/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCUTTER_USE_BUNDLED_RIZIN=off
+      - -DCMAKE_BUILD_TYPE=Release
```

Comment:
```suggestion
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Reply:
Did not know about this type. I'm retrying the build with it